README.yaml

```diff
@@ -0,0 +1,177 @@
+name: "dev.cloudposse.co"
+
+# Canonical GitHub repo
+github_repo: "cloudposse/dev.cloudposse.co"
+
+# License of this project
+license: "APACHE2"
+
+# Badges to display
+badges:
+  - name: "Codefresh Build Status"
+    image: "https://g.codefresh.io/api/badges/build?repoOwner=cloudposse&repoName=dev.cloudposse.co&branch=master&pipelineName=dev.cloudposse.co&accountName=cloudposse&type=cf-1"
+    url: "https://g.codefresh.io/pipelines/dev.cloudposse.co/builds"
+  - name: "Latest Release"
+    image: "https://img.shields.io/github/release/cloudposse/dev.cloudposse.co.svg"
+    url: "https://github.com/cloudposse/dev.cloudposse.co/releases"
+  - name: "Slack Community"
+    image: "https://slack.cloudposse.com/badge.svg"
+    url: "https://slack.cloudposse.com"
+
+description: |-
+  Terraform/Kubernetes Reference Infrastructure for Cloud Posse Audit Organization in AWS. 
+  
+  This account is suitable for receiving all CloudTrail logs from other accounts. 
```

Comment:
This account is suitable for developers to provision and test their own infrastructure